### PR TITLE
Add README doc section of multi-gpu acceleration.

### LINKED ISSUE #278: Add multi-gpu profiling results to REAME doc for DS2.

### PR DESCRIPTION
**Pull request by xinghai-sun**
Resolve #278

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -466,9 +466,21 @@ Test Set                | Aishell Model     | Internal Mandarin Model
 Aishell-Test            |   X.X             |   X.X
 Baidu-Mandarin-Test     |   X.X             |   X.X
 
-#### Multiple GPU Efficiency
+#### Acceleration with Multi-GPUs
+
+We compare the training time with 1, 2, 4, 8, 16 K40 GPUs (with a subset of LibriSpeech samples whose audio durations are between 6.0 and 7.0).  And it shows that a **near-linear** acceleration with multiple GPUs has been achieved. In the following figure, the time (in seconds) used for training is plotted on the blue bars.
```
Comment (pkuyym) [inline deep_speech_2/README.md]: K40m?

Comment (xinghai-sun) [reply deep_speech_2/README.md]: --> Tesla K40m

**Review by pkuyym — APPROVED** [review]
LGTM

File: deep_speech_2/README.md [hunk 2]
```diff
@@ -466,9 +466,21 @@ Test Set                | Aishell Model     | Internal Mandarin Model
 Aishell-Test            |   X.X             |   X.X
 Baidu-Mandarin-Test     |   X.X             |   X.X
 
-#### Multiple GPU Efficiency
+#### Acceleration with Multi-GPUs
+
+We compare the training time with 1, 2, 4, 8, 16 Tesla K40m GPUs (with a subset of LibriSpeech samples whose audio durations are between 6.0 and 7.0).  And it shows that a **near-linear** acceleration with multiple GPUs has been achieved. In the following figure, the time (in seconds) used for training is plotted on the blue bars.
```
Comment (kuke) [inline deep_speech_2/README.md]: 6.0 and 7.0, what is the unit?

Comment (xinghai-sun) [reply deep_speech_2/README.md]: Done.

**Pull request merged**